src/lucky_web/page.cr

```diff
@@ -39,4 +39,16 @@ module LuckyWeb::Page
 
     generate_initializer
   end
+
+  macro p(_arg, **args)
+    def p(_arg, **args)
```

Comment:
I *think* this will still work if you remove the `def p` and just have the macro. Could you give that a shot?

Reply:
Sure!

Reply:
@paulcsmith It did work. I don't know why, since https://crystal-lang.org/docs/syntax_and_semantics/macros.html tells me to include the `def p` but what do *I* know, right? 💃